src/log_unstable.rs

```diff
@@ -31,6 +32,9 @@ pub struct Unstable {
     /// All entries that have not yet been written to storage.
     pub entries: Vec<Entry>,
 
+    /// The size of entries
+    pub entries_size: usize,
```

Comment:
Should be private.

Reply:
It's the same as other variables.

Reply:
We probably should also make other fields private. Prevent another pub field to not make access control worse.

Reply:
I agree. I think most of the public fields can be private. We should clean them up afterward.